Retier CRIB, proxy, adv stocking hatch

serenibyss opened the pull request:
Retiers Crafting Input Buffer, Crafting Input Proxy, and Advanced Stocking Input Hatch as discussed in https://discord.com/channels/181078474394566657/1326201379265449995.

Moves all parts to Space Assembler as well, since late game (UHV+) AE2 parts always go in space assembler now, these recipes were just old and now is a good time to move them.

Advanced Stocking Input Hatch: UEV -> UHV
Crafting Input Buffer: UIV -> UEV
Crafting Input Proxy: UV -> UIV

GT5u PR changing the casing textures on these parts to the new tiers: https://github.com/GTNewHorizons/GT5-Unofficial/pull/3766

|                     **Part**                     | **Old** | **New** |
|:------------------------------------------------:|:-------:|:-------:|
| **Advanced Stocking Input Hatch** _(UEV -> UHV)_ |    ![image](https://github.com/user-attachments/assets/948bd030-02e8-4a90-a665-662ea4a123b5)     |     ![image](https://github.com/user-attachments/assets/54c43d17-4f49-46dd-98b1-1f1a22e997a6)    |
| **Crafting Input Buffer** _(UIV -> UEV)_         |     ![image](https://github.com/user-attachments/assets/5175c8cb-00fe-44cf-a910-ef77edaf543e)    |    ![image](https://github.com/user-attachments/assets/aee639bb-8fc2-4fa1-9497-2c53fc76f72a)     |
| **Crafting Input Proxy** _(UV -> UIV)_           |     ![image](https://github.com/user-attachments/assets/41ab829f-ef11-4761-86d4-42c0279914eb)    |    ![image](https://github.com/user-attachments/assets/d5b93bed-eabe-49c5-ac7f-c4eaf78cd238)     |

miaowwwwww commented:
For Crafting Input Proxy, i feel the AE storage component is unnecessary, They do not increase the difficulty of making the block since it is tiered uiv. we already have storage in the crib in the recipe, i see no reason to have more storage since they do not store more stuff.

boubou19 commented:
As someone pointed out on the discord, we must rework the cribus take regular stocking, so people don't think cribus is an "useless" upgrade to advanced stocking hatch, but a different block for different usecases, like we intent to.

serenibyss commented:
Latest commit:
- `Crafting Input Bus (ME)`:
  - Changed recipe to take a normal Stocking Input Bus rather than advanced
  - Changed research item from Advanced Stocking Input Bus -> ME Interface (normal Stocking Input Bus is the scan item for Advanced Stocking Input Bus already)
- `Crafting Input Buffer (ME)`:
  - Changed recipe to take a Crafting Input Bus (ME) rather than Advanced Stocking Input Bus, since this makes more sense for what the part does, and the cost/tiering is equivalent
